examples/platform/silabs/efr32/spi_multiplex.h

```diff
@@ -17,49 +17,152 @@
 
 #pragma once
 
+#if defined(EFR32MG24)
 #ifdef __cplusplus
 extern "C" {
 #endif
 #include "FreeRTOS.h"
 #include "semphr.h"
 #include "sl_mx25_flash_shutdown_usart_config.h"
+#if (defined(EFR32MG24) && defined(WF200_WIFI))
 #include "sl_spidrv_exp_config.h"
+#endif /* EFR32MG24 && WF200_WIFI */
+#if (defined(EFR32MG24) && defined(RS911X_WIFI))
+#include "sl_spidrv_eusart_exp_config.h"
+#endif /* EFR32MG24 && RS911X_WIFI */
+#if (defined(EFR32MG24) && defined(WF200_WIFI))
 #include "sl_wfx_host_api.h"
+#endif /* EFR32MG24 && WF200_WIFI */
+#include "em_eusart.h"
 #include "spidrv.h"
 
-#define SL_BIT_RATE_LCD 1100000
-#define SL_BIT_RATE_EXP_HDR 16000000
-#define SL_BIT_RATE_SPI_FLASH 10000000
-#define SL_BIT_RATE_UART_CONSOLE 16000000
+#ifdef WF200_WIFI
+// TODO: Investigate why using SL_SPIDRV_EXP_BITRATE is causing WF200 init failure
+// REF: sl_spidrv_exp_config.h
+#define SL_SPIDRV_EXP_BITRATE_MULTIPLEXED 10000000
+#else
+#define SL_SPIDRV_EXP_BITRATE_MULTIPLEXED SL_SPIDRV_EUSART_EXP_BITRATE
+#endif /* WF200_WIFI */
 
-extern SemaphoreHandle_t spi_sem_sync_hdl;
+#define SL_SPIDRV_LCD_BITRATE 1100000
+#define SL_SPIDRV_MX25_FLASH_BITRATE 16000000
```

Comment:
Pretty sure all of these defines are already defined inside of the Gecko SDK. Should use these instead of redeclaring them here.

Reply:
As discussed, these values are already defined within the GSDK

e.g. for the LCD in `./third_party/silabs//gecko_sdk/hardware/driver/memlcd/src/ls013b7dh06/sl_memlcd_display.h`
```
/* Max supported frequency is 1.1 MHz */
#define SL_MEMLCD_SCLK_FREQ               1100000
```